docs/PowerAuth-2024.07.md

```diff
@@ -0,0 +1,83 @@
+# PowerAuth 2024.07
+
+
+## Migration guides
+
+For updating to 2024.07, please follow these migration guides:
+
+- [PowerAuth Server - Migration from version 1.7.0 to version 1.8.0](https://github.com/wultra/powerauth-server/blob/develop/docs/PowerAuth-Server-1.8.0.md)
+- [PowerAuth Push Server - Migration from version 1.7.0 to version 1.8.0](https://github.com/wultra/powerauth-push-server/blob/develop/docs/PowerAuth-Push-Server-1.8.0.md)
+- [PowerAuth Web Flow - Migration from version 1.7.0 to version 1.8.0](https://github.com/wultra/powerauth-webflow/blob/develop/docs/Web-Flow-1.8.0.md)
+
+
+## Components for version 2024.07
+
+
+### Back-End Applications
+
+| Component              | Application Name             | Version | Description                                                 |
+|------------------------|------------------------------|---------|-------------------------------------------------------------|
+| PowerAuth Server       | `powerauth-java-server.war`  | 1.8.0   | Core back-end component for PowerAuth stack.                |
+| PowerAuth Admin        | `powerauth-admin.war`        | 1.8.0   | Administration console for PowerAuth Server.                |
+| PowerAuth Push Server  | `powerauth-push-server.war`  | 1.8.0   | Simple to deploy push server for APNS and FCM.              |
+| Enrollment Server      | `enrollment-server.war`      | 1.8.0   | Enrollment server for PowerAuth.                            |
+| PowerAuth Web Flow     | `powerauth-webflow.war`      | 1.8.0   | Central web authentication page.                            |
+| PowerAuth Next Step    | `powerauth-next-step.war`    | 1.8.0   | Authorization server used for PowerAuth Web Flow component. |
+| PowerAuth Data Adapter | `powerauth-data-adapter.war` | 1.8.0   | Customization component for PowerAuth Web Flow.             |
+| PowerAuth Tpp Engine   | `powerauth-tpp-engine.war`   | 1.8.0   | Third party provider registry and consent engine.           |
+
+
+### Utilities
+
+| Component                   | Application Name            | Version | Description                                                                       |
+|-----------------------------|-----------------------------|---------|-----------------------------------------------------------------------------------|
+| PowerAuth Command Line Tool | `powerauth-java-cmd.jar`    | 1.8.0   | Command line tool for integration testing.                                        |
+| User Data Store             | `user-data-store.war`       | 1.3.0   | Server component which stores clients personal data securely.                     |
+| Mobile Utility Server       | `mobile-utility-server.war` | 1.8.0   | Server component for dynamic SSL pinning, text localization, and other utilities. |
+| SSL Pinning Tool            | `ssl-pinning-tool.jar`      | 1.8.0   | A command line utility used to sign SSL certificates for dynamic SSL pinning.     |
+
+
+### Mobile Libraries
```

Comment:
- [ ]  Update mobile libraries

@TomasKypta @Hopsaheysa @kober32 Could you please comment or even add suggestions to the PR?

Reply:
I guess the mobile versions should be:
| PA SDK | `com.wultra.android.powerauth:powerauth-sdk` | 1.8.1 |
| React Native | `react-native-powerauth-mobile-sdk` | 2.5.1 |
| mToken SDK iOS | `WultraMobileTokenSDK` | 1.10.0 |
| mToken SDK Android | `com.wultra.android.mtokensdk:mtoken-sdk-android` | 1.11.0 |